jobs/image/cosbeta.env

```diff
@@ -1,3 +1,3 @@
 KUBE_NODE_OS_DISTRIBUTION=gci
-KUBE_GCE_NODE_IMAGE=cos-beta-59-9460-57-0
+KUBE_GCE_NODE_IMAGE=cos-stable-59-9460-64-0
```

Comment:
Not sure how the file is used but now that a stable image is used, the name `cosbeta` looks contradictory.